indico/modules/events/abstracts/controllers/management.py

```diff
@@ -113,24 +113,41 @@ def _process(self):
 class RHManageAbstractReviewing(RHManageAbstractsBase):
     """Configure abstract reviewing."""
 
+    @property
+    def ratings(self):
+        return (AbstractReviewRating.query
+                .join(AbstractReviewRating.review)
+                .join(AbstractReview.abstract)
+                .join(AbstractReviewRating.question)
+                .filter(~Abstract.is_deleted, Abstract.event == self.event,
+                        AbstractReviewQuestion.field_type == 'rating'))
+
+    def _scale_ratings(self, scale_min, scale_max):
+        prev_min = abstracts_reviewing_settings.get(self.event, 'scale_lower')
+        prev_max = abstracts_reviewing_settings.get(self.event, 'scale_upper')
+        if not (scale_min != prev_min or scale_max != prev_max):
```

Comment:
this is much clearer IMO:

```suggestion
        if scale_min == prev_min and scale_max == prev_max:
```